src/components/TokenPriceDisplay.tsx

```
@@ -0,0 +1,248 @@
+  }, [tokenAddress]);
+
+  if (initialLoading) {
+    return <p className="text-white text-lg animate-pulse">Loading token data...</p>;
+  }
+
+  if (error && !data) {
+    return <p className="text-red-500 text-lg">Error: {error}</p>;
+  }
+
+  // Access the price value directly from the raw response structure
+  const price = data && data.value 
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)